### DISCUSSION TITLE
Error when table name has prefix

### DISCUSSION THREAD

**Discussion by dhanangpratama** [spatie/laravel-ciphersweet#12] [Q&A]
I'll try this package and I love it. But, I have a problem when my tables use prefix. 

I get this error message

```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'users.id' in 'where clause'
```

and this is generated query

```sql
select
  *
from
  `tbl_users`
where
  exists (
    select
      1
    from
      `tbl_blind_indexes`
    where
      `indexable_type` = Modules \ User \ Models \ User
      and `indexable_id` = users.id
      and `name` = email_index
      and `value` = 60c6512c33cb21c0c34052638d03a44ef2c1491afcad2d9e13e9fe9c23fecc9b
  )
limit
  1
```

The prefix not included in  where clause (which is: users.id), Is this a bug?

Thank you!

**Comment by andrewdrake** [comment]
I have the same error, yes looks like a bug

**Comment by juandinhozz** [comment]
@andrewdrake  I had the same problem. An additional solution in my project was to modify:


`vendor\spatie\laravel-ciphersweet\src\Concerns\UsesCipherSweet.php
`

Replacing:
`->where('indexable_id', DB::raw($this->getTable() . '.' . $this->getKeyName())))
`

By:
`->where('indexable_id', DB::raw($this->getConnection()->getTablePrefix() . $this->getTable() . '.' . $this->getKeyName())))
`

It worked for me while fixing the bug.
